src/components/Header/NetworkSelector.tsx

```diff
@@ -17,20 +17,21 @@ import { replaceURLParam } from 'utils/routes'
 
 import { useAppDispatch } from '../../state/hooks'
 import { switchToNetwork } from '../../utils/switchToNetwork'
+import { transparentize } from 'polished'
```

Comment:
You should do these changes on the mod file rather than here, no?

Reply:
oh sheit true.